### DISCUSSION TITLE
Add support for system proxy

### DISCUSSION THREAD

**Discussion by LunaticoCR** [Kong/insomnia#5564] [Ideas] [answered]
I would be great to have a "Use system proxy" option in the HTTP Network Proxy section in Insomnia Preferences. I tend to forget to change my proxy setting for Insomnia and scratch my head for a moment wondering why my request aren't working before I remember that I need to select the "Enable proxy" option.

**Comment by phsc84** [comment]
Duplicate of #3590

**Reply by subnetmarco** [reply, marked as answer]
Closing this discussion in favor of https://github.com/Kong/insomnia/discussions/3590
